### ISSUE TITLE
lengthMenu show all

### ISSUE THREAD

**Issue by jfols** [aldeed/meteor-tabular#69]
`lengthMenu: [[10, 25, 50, -1], [10, 25, 50, "All"]]` only shows 1 on the `All` option.
https://datatables.net/examples/advanced_init/length_menu.html

**Comment by kirilan** [comment]
I get the same behaviour. 
the -1 is not interpreted as all but actually as -1; maybe if we can call a function that returns the total number of rows and put it there that will be a workaround but I did not find such a function and way to place it there.

**Comment by landm** [comment]
I have the same issue. DataTables interprets "-1" as turning pagination off entirely. I tried passing the "paging: false" option to the table and it also causes only 1 record to be returned.

**Comment by aldeed** [comment]
Probably needs implementing within the limiting code in this package. Anything related to filter/sort/skip/limit needs specific handling since we're doing just-in-time publishing.

I'll accept a PR or fix this when I have time.

**Comment by gentunian** [comment]
I'm reading the code. I've tested using `limit = 0` while removing the [if condtion](https://github.com/aldeed/meteor-tabular/blob/master/server/tabular.js#L70):

```
  // Verify that limit is not 0, because that will actually
  // publish all document _ids.
  if (limit === 0) {
```

Using `limit = 0` retrieves all documents.

**Comment by gentunian** [comment]
The reason why pagination is not working is because DataTables  `-1` paged value is being wrapped as a [mongodb `limit` option](http://docs.meteor.com/#/full/find). So the find query should be passed `0`  as `limit` option if the user selects `-1` pagination as a DataTable paging option.

**Comment by aldeed** [comment]
Should be fixed in 1.3.0 release
